.gitpod.yml

```diff
@@ -38,5 +39,5 @@ ports:
   - name: App
     port: 5137
```

Comment:
```suggestion
    port: 5173
```

Reply:
@anbraten the app runs in port 3000 though?